packages/third-parties/README.md

```diff
@@ -0,0 +1,51 @@
+# `@next/third-parties`
+
```

Comment:
Let's add a note at the top that says this is still experimental or early in development, just so folks are aware :)